panoptes_aggregation/tests/extractor_tests/test_question_extractor.py

```diff
@@ -121,3 +121,54 @@
     "Test pluck field functionality with a question extractor",
     kwargs={'pluck': single_pluck_keys},
     test_name='TestSinglePluck')
+
+
+feedback_pluck_classification = {
+    "id": 359841171,
+    "annotations": [{
+        "task": "T0",
+        "value": "0",
+        "taskType": "single"
+    }],
+    "metadata": {
+        "feedback": {
+            "T0": [
+                {
+                    "answer": "0",
+                    "success": True,
+                    "strategy": "singleAnswerQuestion"
+                }
+            ]
+        }
+    },
+    "subject": {
+        "id": 68561182,
+        "metadata": {
+            "filename": "uds_H.gf19767.png",
+        }
+    }
+}
+
+feedback_pluck_keys = {
+    "feedback": "metadata.feedback.T0"
+}
+
+feedback_pluck_expected = {
+    "0": 1,
+    "feedback": {
+        "agreement_score": 1.0,
+        "success": [
+            True
+        ],
+        "true_answer": [
+            "0"
+        ]
+    }
+}
+
+TestFeedbackPluck = ExtractorTest(extractors.question_extractor,
+    feedback_pluck_classification,
+    feedback_pluck_expected,
+    "Test pluck field functionality for feedback data with a question extractor",
+    kwargs={'pluck': feedback_pluck_keys},
+    test_name='TestFeedbackPluck')
```

Comment:
There should be additional tests where the `get_feedback_function` will return `None` (e.g. the feedback dictionary on the classification is empty and when it is not a list).

Reply:
Also, the format should be:
```python
TestFeedbackPluck = ExtractorTest(
    extractors.question_extractor,
    feedback_pluck_classification,
    feedback_pluck_expected,
    "Test pluck field functionality for feedback data with a question extractor",
    kwargs={'pluck': feedback_pluck_keys},
    test_name='TestFeedbackPluck'
)
```

Note that there are no args on the first line and the closing bracket is not indented and on its own line.